utils/record/record.go

```diff
@@ -207,6 +239,13 @@ func NewAPIFactorySettings() api.Settings {
 // Send notifications for triggered event if user is subscribed
 func (e *EventRecorderAdapter) sendNotifications(object runtime.Object, opts EventOptions) error {
 	logCtx := logutil.WithObject(object)
+	_, namespace, name := logutil.KindNamespaceName(logCtx)
+	startTime := timeutil.Now()
+	defer func() {
+		duration := time.Since(startTime)
+		e.NotificationSendPerformance.WithLabelValues(namespace, name).Observe(duration.Seconds())
+		logCtx.WithField("time_ms", duration.Seconds()*1e3).Info("Notification sent")
```

Comment:
`Info` might be too verbose. Could you please change it to `Debug`?

Reply:
Thanks fixed the comments.